docs/source/contribution.rst

```diff
@@ -443,3 +291,110 @@ Open ``index.html`` with the browser and see if it is rendered as expected.
 
    Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
    If you modified docstrings, make sure to install the module (e.g., using `pip install -e .`) before building the documentation.
+
+.. note::
+
+   If you are unsure about how to write the documentation or failt to build it locally, you can submit a PR without documentations.
+   Reviewers will help you with it.
+
+Other Forms of Contribution
+---------------------------
+
+There are several other ways in which you can contribute to Chainer without directly working with the code base.
+Following are such contributions.
+
+* Sending a question/reply to `StackOverflow <https://stackoverflow.com/>`_ (with ``chainer`` tag) or `Chainer User Group <https://groups.google.com/forum/#!forum/chainer>`_
+* Open-sourcing an external example
+* Writing a post about Chainer
+
+Development Cycle
+-----------------
+
+This section explains the development process of Chainer.
+
+Versioning
+~~~~~~~~~~
+
+The versioning of Chainer follows `PEP 440 <https://www.python.org/dev/peps/pep-0440/>`_ and a part of `Semantic versioning <https://semver.org/>`_.
+The version number consists of three or four parts: ``X.Y.Zw`` where ``X`` denotes the **major version**, ``Y`` denotes the **minor version**, ``Z`` denotes the **revision number**, and the optional ``w`` denotes the pre-release suffix.
+While the major, minor, and revision numbers follow the rule of semantic versioning, the pre-release suffix follows PEP 440, the Python community standards.
+
+**Note that a major update basically does not contain compatibility-breaking changes from the last release candidate (RC).**
+This is not a strict rule, though; if there is a critical bug in the API that need to be fixed for the major version, breaking changes may be introduced.
+
+For more on backward compatibility, please refer to the :ref:`compatibility`.
+
+.. _contrib-release-cycle:
+
+Release Cycle
+~~~~~~~~~~~~~
```

Comment:
We need to revisit this section; currently we are not following this 3-month cycle.

Reply:
How would you like to change it?

Reply:
- stable versions are a series of minor updates (can be revision one, but not so usually)
- we may have more than one beta release (fix the table or think about another way to explain the parallel schedule)

Reply:
Thanks for the clarification. I made some changes as follows.

> - stable versions are a series of minor updates (can be revision one, but not so usually)

Fixed.

> - we may have more than one beta release (fix the table or think about another way to explain the parallel schedule)

Tried to explain it in the following body and wrote right before the table that this is an example and thus isn't always exact.